Util/loadTimeSeriesSampleTemplate.m

```diff
@@ -0,0 +1,101 @@
+function [coordinates, dimensions, variables] = loadTimeSeriesSampleTemplate(timev, latd, lond, depthd)
+% function [coordinates, dimensions, variables] = loadTimeSeriesSampleTemplate(timev, latd, lond, depthd)
+%
+% Load the default timeseries coordinates string and dimensions/variables structures
+%
+% Inputs:
+%
+% timev - the time values
+%
+% latd - the latitude dimension index list
+%        default: []
+% lond - the longitude dimension index list
+%        default: []
+% depthd - the depth dimension index list
+%        default: []
+%
+% Outputs:
+%
+% coordinates - a string with all coordinates names
+% dimensions - a structure with all dimensions prefilled
+% variables - a structure with all variables prefilled
+%
+% Example:
+% [c,d,v] = loadTimeSeriesSampleTemplate([1,2,3])
+% % check coordinates
+% assert(strcmpi(,coordinates,'TIME LATITUDE LONGITUDE NOMINAL_DEPTH'))
```

Comment:
something is missing here

Reply:
Sorry - clearly I didn't check the docstring - its a mix of matlab&python! Had to wrap isequal around the equality testing, move len-> length. I also change for more explicit names.